db/seeds.rb

```diff
@@ -1,7 +1,172 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+# https://github.com/pejotrich/flatmate
+# https://github.com/andrewbonas/rails_facebook
 #
-# Examples:
 #
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+require "open-uri"
+require 'json'
+
+puts "--- GAME  START ---"
+puts "---"
+puts "---"
+puts "cleaning house :)"
+User.destroy_all
+# Bike.destroy_all
+# Booking.destroy_all
+# Review.destroy_all
+# Favourite.destroy_all
+
+# # Kills all Active storage items ##
+ActiveStorage::Attachment.all.each { |attachment| attachment.purge }
+
+puts "done cleaning house.."
+
+puts "-- Making Humans"
+## kill all images in all models ##
+
+## User Master OverLoad # HOLD GME !! ##
+1.times do
+  file = File.open("./db/avatars/00.jpeg")
+
+  make_me = User.create!(
+    first_name: "Jullian",
+    last_name: "Daddy",
+    git_user: "",
+    bio: "$GME goes BRRRRRRRR 💎🙌",
+    location: "Melbourne",
+    email: "spin@gmail.com",
+    admin: true,
+    password: "123456"
+  )
+  make_me.avatar.attach(io: file, filename: "#{make_me.first_name}.jpeg", content_type: 'image/jpeg')
+  puts "made #{make_me.first_name} #{make_me.last_name}"
+end
+
+## User Standard User ##
+1.times do
+  file = File.open("./db/avatars/22.png")
+
+  make_me = User.create!(
+    first_name: "Standard",
+    last_name: "User",
+    bio: "I'm user a standard user, looking at a Dev..",
+    location: "Melbourne",
+    email: "user@gmail.com",
+    admin: false,
+    password: "123456"
+  )
+  make_me.avatar.attach(io: file, filename: "#{make_me.first_name}.png", content_type: 'image/png')
+  puts "made #{make_me.first_name} #{make_me.last_name}"
+end
+## Make the Plebs ##
+count = 0
+
+11.times do
+  file = File.open("./db/avatars/#{count}.jpeg")
+
+  ## make the instance
+  make_me = User.create!(
+    first_name: Faker::Name.first_name,
+    last_name: Faker::Name.last_name,
+    bio: Faker::TvShows::Buffy.quote,
```

Comment:
🚫  Buffy quotes 😂

Reply:
yeah i"m going to pull it all from the git hub api but I'll sort that out tonight 
we need to migrate out the models also for the green features